updated dependencies, modified small things to the code to work with …

mpwsh opened the pull request:
Hi Matthias! I was playing with your rustly example and found that i was not being able to compile.
i updated the dependencies and changed little things to make it work.
Thanks for the blog post, really informative

mpwsh committed:
updated dependencies, modified small things to the code to work with new versions

mpwsh committed:
added dockerfile

mre commented:
Uh that looks very clean! Thanks so much @marianopw.
You even added a `Dockerfile`, which is a great little extra.

Did you try to run it on stable Rust. Got word that there were some recent Rust changes that would make it possible.
Also, would you like to add yourself to the list of authors? It would be nice to co-maintain this with you.

mpwsh committed:
fixed dockerfile, added index

mpwsh commented:
I tried using stable but received the following error:
Error: Pear requires a 'dev' or 'nightly' version of rustc.

I've updated the Dockerfile and added an index page showing the usage of rustly.
Glad you found the commit useful! 
I will be adding some more functionality to rustly in a few weeks, maybe i'll add myself to the authors then :) 
Will try to check the repo in the future to see if it still compiles.

Thanks again !

mre commented:
Thanks, merged. Looking forward to your future contributions. 😊 
FYI: I've updated [the blog post](https://endler.dev/2017/rust-url-shortener/) a bit (just the wording, not the code yet 😉).
